tests/test_column.py

```diff
@@ -115,6 +115,44 @@ def test_filewise(num_files, values):
         column.set([], index=index)
 
 
+def test_get_as_segmented():
+
+    db = pytest.DB
+    db._files_duration = {}
```

Comment:
Why do you have to set this, shouldn't there be a default value?

Reply:
The same comment holds for the other places in the tests where this is explicitly set.

Reply:
We don't have to set it, I just wanted to make sure it's used. Should I add a separate test instead?

Reply:
Ah wait a moment. I misunderstood your question. We have to reset it since `pytest.DB` is shared by several tests and we always want to start from an empty dictionary.

Reply:
To make sure I get this correctly: it's default value is `None` and then it will not be used inside the tests. You force it to be used by setting it to `{}`.

Maybe it will be easier to understand if we make it an argument to the test or add a separate test.
I would test for:
* not setting it
* setting it to `{}`
* setting a different file duration then the actual one for a given file and test if this was used afterwards

Reply:
The default value is `{}`.

Reply:
But since we share `pytest.DB` we have to reset it manually to start from an empty dictionary again.

Reply:
Ah, ok, then please add a comment why you are resetting.

On top of that I think it would still be good to add a test with `db._files_duration` containing an entry and to check that this entry is used when setting `end`

Reply:
> Ah, ok, then please add a comment why you are resetting.

I will remove the line, since we already reset at the end of each function and there is already a comment

> On top of that I think it would still be good to add a test with db._files_duration containing an entry and to check that this entry is used when setting end

done, I also check now that non-existing entries are added

Reply:
Very cool, thanks.